README.md

```diff
@@ -2,117 +2,83 @@
 
 ## Usage
 
-### Pre Requisites
+### Prerequisites
 
-Before running any command, you need to create a `.env` file and set a BIP-39 compatible mnemonic as an environment
-variable. Follow the example in `.env.example`. If you don't already have a mnemonic, use this [website](https://iancoleman.io/bip39/) to generate one.
+- NodeJS - 12.x
 
-Then, proceed with installing dependencies:
+- Solc - v0.8.13 (https://github.com/ethereum/solidity/releases/tag/v0.8.13)
 
-```sh
-$ yarn install
-```
-
-### Compile
+### Installing
 
-Compile the smart contracts with Hardhat:
-
-```sh
-$ yarn compile
 ```
 
-### TypeChain
-
-Compile the smart contracts and generate TypeChain artifacts:
+yarn install
 
-```sh
-$ yarn typechain
 ```
 
-### Linting and code formatting
```

Comment:
I think we should keep this section on the commands required to detect and fix lint issues. We use these command to fix the lint issues.

Reply:
Right but this part is already covered in hardhat commands [here](https://github.com/VenusProtocol/oracle/pull/54/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44), should we need yarn commands too?

Reply:
Do hardhat commands run the linter?

Reply:
Yes they do !

Reply:
The `yarn` commands looks simple and are easier to use. I would prefer to keep them if you guys agree. 

Or else we can create a specific section on how to fix the ts and sol lint issues when pipeline fails due to lint checks. And we can keep the linting specific npx command ordered there.